tidysic/audio_file.py

```diff
@@ -95,10 +95,11 @@ def guess_tags(self, dry_run):
                     answer = input('(r/d) ? ')
 
                 if answer == 'r':
+                    new_tags = {}
```

Comment:
Also, `new_tags` is not defined in this branching, thus key/value dict cannot be assigned like that.

Reply:
Good eye ! Thanks for the fix.